README.md

```diff
@@ -1191,6 +1191,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [KooZic](https://koozic.net/) - Music server with powerful playlist features and Subsonic compatibility. ([Demo](https://demo.koozic.net/public), [Source Code](https://github.com/DocMarty84/koozic)) `LGPL-3.0/MIT` `Python`
 - [LibreTime](https://libretime.org) - Simple, open source platform that lets you broadcast streaming radio on the web (fork of [Airtime](https://github.com/sourcefabric/Airtime)). ([Source Code](https://github.com/LibreTime/libretime)) `AGPL-3.0` `PHP`
 - [LMS](https://github.com/epoupon/lms) - Access your self-hosted music using a web interface. `GPL-3.0` `C++`
+- [Maloja](https://github.com/krateng/maloja) - Self-hosted music scrobble database, alternative to last.fm. ([Demo](https://maloja.krateng.ch/)) `GPL-3.0` `Python`
```

Comment:
>https://github.com/krateng/maloja#docker--podman
- Please change the language tag to `Python/Docker`

Reply:
Done!